crates/op-rbuilder/README.md

```diff
@@ -49,14 +49,20 @@ op-rbuilder has an integration test framework that runs the builder against mock
 To run the integration tests, run:
 
 ```bash
+# Ensure you have op-reth installed in your path,
+# you can download it with the command below and move it to a locaiton in your path
+./scripts/ci/download-op-reth.sh
```

Comment:
this would only download for linux systems? wouldn't work for osx right?

Reply:
oh I see you updated the script

Reply:
Happy to keep the script as is, just figured it would be useful to have something that worked locally and on CI